ios/Podfile.lock

```diff
@@ -1,82 +0,0 @@
-PODS:
```

Comment:
From Cocoapods recommendations, we should keep this file in the repository.
https://github.com/flutter/flutter/issues/9299#issuecomment-295221934

`The reason for checking the Podfile.lock in is so the app developer can exactly recreate and share a Pod configuration across a team. Without checking in the Podfile.lock, developers on the same team will get different versions of pods depending upon when they do pod install.`